src/fund_my_watcard/main.py

```diff
@@ -128,5 +135,58 @@ def main():
         if query_yes_no(WILL_RESET_YOUR_CONFIG_FILE_WARNING, "no"):
             reset_config_file()
 
+    elif args.valid:
+        if os.path.isfile(CONFIG_FILE_PATH):
+            with open(CONFIG_FILE_PATH) as f:
+                _config = json.load(f)
+        else:
+            report_error(CAN_NOT_FIND_CONFIG_FILE)
+
+        if _config["encrypted"] == "True":
+            report_warning(CONFIG_FILE_HAS_BEEN_ENCRYPTED)
+        else:
+            if re.fullmatch('[!@#$%^&*(),.?":{}|<>]', _config["userName"]):
+                report_error("userName has special characters, please remove them.")
+
+            if re.fullmatch("([a-zA-Z]+ +)", _config["ordName"]):
```

Comment:
Why there are two "+" here?

Reply:
When I wrote the regex I added an extra `+` to make it capture one or more space characters for names like `First Middle Last` but we can remove that extra `+` since the regex can just capture `First ` and that would make the name valid.